public/javascripts/services.js

```diff
@@ -215,6 +215,16 @@ services.factory('RepFactory', function($resource) {
 });
 
 /**
+ * This is the factory used to update a representative based on a given
+ * Buyer and Representative ID
+ */
+services.factory('RepEditFactory', function($resource) {
```

Comment:
@nick22891 Notice I don't have any "EditFactories" in my code?  I try my best to have just one factory that can do it all.  For instance:
```javascript
services.factory('BuyerFactory', function($resource) {
    return $resource('/buyer/:id', {}, {
        show: { method: 'GET'},
        create: { method: 'POST'},
        update: { method: 'PUT', params: {id: '@id'}}
    });
});
```

Attempt to have just one factory that can create, update and get information from the web-services.